x-pack/plugins/apm/server/lib/transactions/get_transaction/index.ts

```diff
@@ -36,7 +40,7 @@ export async function getTransaction({
           filter: asMutableArray([
             { term: { [TRANSACTION_ID]: transactionId } },
             ...(traceId ? [{ term: { [TRACE_ID]: traceId } }] : []),
-            ...('start' in setup ? rangeQuery(setup.start, setup.end) : []),
+            ...('start' in setup ? rangeQuery(start, end) : []),
```

Comment:
`start` is no longer on setup:

```suggestion
            ...(start && end ? rangeQuery(start, end) : []),
```